pkg/resolver/searchSchema.go

```diff
@@ -0,0 +1,83 @@
+package resolver
+
+import (
+	"context"
+
+	"github.com/doug-martin/goqu/v9"
+	"github.com/driftprogramming/pgxpoolmock"
+	db "github.com/stolostron/search-v2-api/pkg/database"
+	klog "k8s.io/klog/v2"
+)
+
+type SearchSchemaMessage struct {
+	pool   pgxpoolmock.PgxPool
+	query  string
+	params []interface{}
+}
```

Comment:
This combines SearchSchema and Messages into a single struct. We should separate for clarity and also to allow for more messages in the future.